### PR TITLE
Formatações, tutorial pra instalar pip, opiniões

### PR DESCRIPTION
**Pull request by iuriguilherme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,91 +170,104 @@ Esse superusuário que você vai usar para entrar localmente na interface admini
...
 
 [Instruções para o PythonAnywhere](/deploy%20python%20anywhere.md)
 
+---
+
 ### Remoto, no CodeAnywhere:
 
 Faça os procedimentos descritos acima, clonando o repositório, criando virtualenv, criando banco de dados e superusuário.
 
+Edite o arquivo `site/emaula/website/settings/dev.py` para colocar o endereço do seu container em **ALLOWED_HOSTS**, por exemplo:
```
Comment (rsip22) [inline README.md]:
```suggestion
Edite o arquivo `site/emaula/website/settings.py` para colocar o endereço do seu container em **ALLOWED_HOSTS**, por exemplo:
```

File: README.md [hunk 2]
```diff
@@ -103,91 +170,104 @@ Esse superusuário que você vai usar para entrar localmente na interface admini
...
-Edite o arquivo site/emaula/website/settings/dev.py para colocar o endereço do seu container em ALLOWED_HOSTS, por exemplo:
-```
+```py
 ALLOWED_HOSTS = ['emaula-rsip22207208.codeanyapp.com']
 ```
 
 Inicialize o servidor com as configurações de desenvolvimento e com a porta 3000, para poder acessar a app por HTTPS:
 
+```sh
+$ python3 manage.py runserver 0.0.0.0:3000 --settings=website.settings.dev
```
Comment (rsip22) [inline README.md]:
```suggestion
$ python3 manage.py runserver 0.0.0.0:3000 --settings=website.settings
```

**Review by rsip22 — COMMENTED** [review]
Since you are editing, would you mind fixing these two  things as well?

**Comment by iuriguilherme** [comment]
> do you mind

I do.

**Comment by rsip22** [comment]
Thank you for your contribution, @iuriguilherme !

**Pull request merged**